src/goto-programs/remove_skip.cpp

```diff
@@ -12,7 +12,10 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "remove_skip.h"
 #include "goto_model.h"
 
-bool is_skip(const goto_programt &body, goto_programt::const_targett it)
+bool is_skip(
```

Comment:
Please document the new param at least

Reply:
Thank you for making me do this. It helped me see that I hadn't actually implemented this new feature... oO - fixed now.